### DISCUSSION TITLE
Is it preferrable to start taking part in audit contests or should i wait till completion of the course.

### DISCUSSION THREAD

**Discussion by Debarunn1806** [Cyfrin/security-and-auditing-full-course-s23#147] [General]
hello everyone ,
I have started the smart contract auditing course and am curious that how should i apply my learnings to real world applications ...Any suggestions

**Comment by EngrPips** [comment]
I would say maybe join hackhatons, start applying for jobs or contribute to real world codebase

**Comment by alymurtazamemon** [comment]
@Debarunn1806 You can start with the First Flight competitions these are beginner-friendly. Then you just need to participate in the audit contests (start with small codebases) and learn from each contest. At the end of the contest check what you missed and why you missed that and then just work on your weak topics. This way you will improve after each contest and one day you will perform better in audit contests.
